### ISSUE TITLE
Mobile controls

### ISSUE THREAD

**Issue by Takrak** [nbarkhina/N64Wasm#26]
Occasionally the virtual stick will get stuck in one position after flicking it a bit and will not be able to be moved until you reload the page additionally mobile is missing a few buttons that are needed in some games (all the c buttons, both l and r, and the d pad) I don't know if those are available but just off by default but either way the layout of the buttons should be able to be changed from the menu when in a game.

**Comment by nbarkhina** [comment]
Yeah I've seen the "virtual stick getting stuck" issue as well, I think it is related to NippleJS which is the library I am using and is probably this issue which has not been fixed yet:

https://github.com/yoannmoinet/nipplejs/issues/151

In terms of the mobile buttons missing, I totally agree and it was mainly that there just isn't a lot of space for all the buttons in portrait mode (which is the only mode that is currently supported). So it was really just a tradeoff where I kind of picked the most common buttons I thought people would need (understandably it's not a solution but that was my thinking) The ways to solve that long term would be either to create a landscape mode where I would have buttons overlayed, or maybe make the buttons configurable like you said.
